assets/js/blocks/cart-checkout-shared/payment-methods/style.scss

```diff
@@ -198,7 +198,11 @@
 
 	.wc-block-components-radio-control-accordion-option,
 	.wc-block-components-radio-control__option {
-		@include with-translucent-border(1px 1px 0 1px);
+		border: 1px solid $universal-border-light;
```

Comment:
This one had no bottom border previously—was this change intentional?

Reply:
Yes, this change was intentional. Previously, the borders were added on pseudo elements.